src/Symfony/Component/Form/Extension/Core/Type/TimezoneType.php

```diff
@@ -37,7 +37,7 @@ public function configureOptions(OptionsResolver $resolver)
     {
         $resolver->setDefaults(array(
             'choice_loader' => function (Options $options) {
-                $regions = $options['regions'];
+                $regions = $options->offsetGet('regions', false);
```

Comment:
After https://github.com/symfony/symfony/pull/28878, the way to opt-out of the deprecation is:
```php
'choice_loader' => function (Options $options) {
    return new CallbackChoiceLoader(function () use ($options) {
        return getTimezones($options['regions']);
    });
},
```

Reply:
that's a bit cheating and may not always be possible to shift the scope

Reply:
Yeah, I meant here :) for this case.

Reply:
well.. we're still consuming a deprecated option :dagger: 

it should trigger IMHO, unless explicitly opt-out

Reply:
https://github.com/symfony/symfony/pull/28878 updated to trigger each time it's evaluated. I'm still thinking about the way to opt-out of the deprecation.

Reply:
`offsetGet($option/*, bool $triggerDeprecation = true*/)` seems fine? That be the easiest :)